prove_stuffs/prove_helpers.h

```diff
@@ -12,20 +12,11 @@
 // OPT_ADD automatically add the content of "a" if the next argument of the variadic macro is set
 #define OPT_ADD(a, ...) ELEVENTH_ARGUMENT(dummy, ## __VA_ARGS__, a, a, a, a, a, a, a, a, a, , dummy)
 
-#define GEN_ASSERT(attr, attr_code, attr_length, attr_flags, ...) \
-    p_assert(                                                  \
-            ((attr)->code != (attr_code)) || \
-            (                                                \
-               ((attr)->length == (attr_length)) && \
-               ((attr)->flags == (attr_flags))               \
-               OPT_ADD(&&(__VA_ARGS__), ##__VA_ARGS__)\
-            ) \
-          )
-
 #define BUF_CHECK_LENGTH(buf, length, offset) \
     (((length) <= 255 && (buf)[offset] == (length)) || \
     ((length) > 255 && (*(uint16_t *) ((buf)+(offset))) == (length)))
 
+#ifdef PROVERS_SEAHORN
```

Comment:
Are you sure this does not break "non verifier" build ? What if I included `BUF_GEN_ASSERT` in my code and I want to compile into eBPF ?

Reply:
Yes, there is an #else that defines empty macros

Reply:
lgtm then